### PR TITLE
Some cleanups found when testing warnings.

### PR DESCRIPTION
**Pull request by Kangz**
This is to trigger a travis build, please do not land.

### CODE REVIEW

**Comment by Kangz** [comment]
PTAL!

File: src/backend/metal/CommandBufferMTL.mm [hunk 1]
```diff
@@ -274,10 +273,6 @@ void EndRenderPass() {
                 case Command::SetPushConstants:
                     {
                         SetPushConstantsCmd* cmd = commands.NextCommand<SetPushConstantsCmd>();
-                        uint32_t* valuesUInt = commands.NextData<uint32_t>(cmd->count);
```
Comment (kainino0x) [inline src/backend/metal/CommandBufferMTL.mm]: Isn't this `commands.NextData` call necessary even if the result isn't used?

Comment (Kangz) [inline src/backend/metal/CommandBufferMTL.mm]: Good catch, thanks

**Comment by Kangz** [comment]
Thanks!

**Pull request merged**